ansible_community_steering_committee.rst

```diff
@@ -0,0 +1,95 @@
+- Ballot on approval changes to policy and new proposals.
+- Identify and train individuals from the community who are interested in contributing to Ansible.
+- Review collections submitted for inclusion in the Ansible package and decide whether include them or not.
+
+Members:
+========
+
+*Key: Full name (IRC/Matrix Nickname)*
+
+1. Felix Fontein (``felixfontein``)
+2. Tadej Borovšak (``tadeboro``)
+3. James Cassell (``cyberpear``)
+4. John Barker (``gundalow``)
+5. Andrew Klychkov (``andersson007_``)
+6. Alicia Cozine (``acozine``)
+7. Sorin Sbarnea (``zbr``)
+8. Jill Rouleau (``jillr``)
+9. Brad Thornton (``cidrblock``)
+10. Dylan Silva (``thaumos``)
+
+-  Currently, all the Steering Committee members are selected based on
+   their active contribution to the Ansible Project.
+-  In future, the selection process will be based on the election model
+   through community.
+
+New Policy/Proposals & Inclusion Requests:
+==========================================
+
+Requests regarding new collection inclusion requests should be submitted
+via this `GitHub discussion <https://github.com/ansible-collections/ansible-inclusion/discussions/new>`_.
+
+Requests regarding new inclusion policy/proposals should be submitted through a new pull request to `ansible-collections/overview <https://github.com/ansible-collections/overview>`_ GitHub repository.
+
+Also, we earlier used a `Community Working Group Meeting Agenda <https://github.com/ansible/community/issues/539>`_ to track
+agenda topics that needed to be discussed in the weekly Ansible Community `IRC/Matrix <https://docs.ansible.com/ansible/devel/community/communication.html#real-time-chat>`_ meetings. Now, to categorize each topic separately we’ve
+created a new GitHub repo `community-topics <https://github.com/ansible-community/community-topics>`_.
+If you want to discuss an idea that can impact the community, collection best practices, inclusion policy, submitted new Policy/Proposals & New Collection Inclusion Requests or other topics the steering committee is responsible for, create a new issue in the `community-topics repository <https://github.com/ansible-community/community-topics>`_ as a discussion topic, and it’ll be discussed publicly in the weekly community `IRC/Matrix <https://docs.ansible.com/ansible/devel/community/communication.html#real-time-chat>`_ meeting and/or asynchronously in the issue itself.
+
+After every meeting, the meeting minutes/summary and meeting logs are posted to the `original issue <https://github.com/ansible/community/issues/539>`_.
+
+If you cannot join the meeting or you are in another time zone then you can
+directly use GitHub Discussions feature to discuss your new
+proposals/collection inclusion requests and the steering committee will
+discuss the final conclusions during the weekly `IRC/Matrix <https://docs.ansible.com/ansible/devel/community/communication.html#real-time-chat>`_ meeting and then will vote on them.
```

Comment:
I'm not sure how much sense it makes to document that now since we're in the process of stopping to vote in the meetings and only vote asynchronously. I don't think we plan to have any more votes in the meeting at all.

CC @tadeboro

Reply:
We could remove this, add `FIXME ...`, and elaborate later after the official decision about moving to async discussion / voting is made.

Reply:
I agree with @felixfontein here. We have already effectively stopped voting during meetings (we direct people to vote in the issue), so there is no need to document something that we know is on its deathbed.

Reply:
Also imo it makes sense to mention the labels that we use. We can achieve this with:
1. Listing them here with short explanation (especially `next_meeting` and `active-vote`, thanks @felixfontein for the idea).
2.  Or we could list them in the community-topic's README and mention the labels here pointing to the README for details.
3. Maybe we shouldn't mention them anywhere at all:)

What do you folks think?

Reply:
@Ompragash we might not even discuss it during a meeting. It could happen that an issue in community-topics is discussed completely asynchronously and never mentioned in a meeting.

Reply:
@Andersson007 I think we can actually slowly retire the `next_meeting` label now that we are going async.

Reply:
@felixfontein agreed. Though maybe there will be special cases that we will want to discuss in real time

Reply:
so I wouldn't rush:) or maybe replace it with something else like `meeting_discussion`

Reply:
@Andersson007 I would also mention the [Project Board](https://github.com/orgs/ansible-community/projects/2/views/5) that Greg has created and we can directly use them instead of labels. It's more effective than labels in terms of categorizing the topics and all we have to do is change the status of the issue topic on the right pane like below:
![Screenshot 2022-01-05 at 6 26 23 PM](https://user-images.githubusercontent.com/21008429/148221140-2b91d424-008a-4ea1-a887-e79469ebc171.png)

Reply:
@Ompragash i would leave labels for now:) they can be helpful for us and for occasional people